build.gradle

```diff
@@ -63,9 +60,11 @@ ext {
             '-DCMAKE_SKIP_RPATH=FALSE',
             '-DCMAKE_BUILD_RPATH_USE_ORIGIN=TRUE',
             '-DBUILD_opencv_apps=OFF',
+            '-DBUILD_opencv_js_bindings_generator=OFF',
+            '-DBUILD_opencv_python_bindings_generator=OFF',
+            '-DBUILD_opencv_python_tests=OFF',
```

Comment:
Why the changes to Python?

Reply:
I thought when I looked at the Mac builds, they included Python code that wasn't supposed to be there. Seems like that isn't the case, since the runners don't even have the necessary dependencies to build Python libraries. Change has been reverted since it's unnecessary.